pkg/pillar/hypervisor/xen.go

```diff
@@ -218,10 +218,13 @@ func (ctx xenContext) CreateDomConfig(domainName string, config types.DomainConf
 
 	var diskStrings []string
 	var p9Strings []string
+	var volumes []string
 	for i, ds := range diskStatusList {
-		if ds.Format == zconfig.Format_CONTAINER {
+		if ds.Format == zconfig.Format_CONTAINER && ds.MountDir != "" {
```

Comment:
I am confused by this. If I read this correctly, then it only works if the mount dir is not blank. What happens with the root directory? Or are we now requiring that _every_ mounted directory have a mountdir, including root, which would be `/`? If that is it, I actually kind of like that. No implicit anything.

Reply:
>What happens with the root directory?

We are setting up mount point in domainmgr for root directory.

Reply:
So whereas when it was just one disk, it was implicit to be `/`, but now it will be explicit? I like it.